### ISSUE TITLE
Using of JQ leads to the loss on int64 numbers

### ISSUE THREAD

**Issue by petli-openshift** [openshift-online/ocm-cli#295]
I am working to add a new API that one of the field is int64. I noticed there's loss using `ocm` v.s. using `curl`,
```
echo '{"type":"sca", "arch":"x86_64"}' | ocm post http://localhost:8500/api/accounts_mgmt/v1/certificates
------ response ------
"id": 2377845297775806000,

curl http://localhost:8500/api/accounts_mgmt/v1/certificates -d '{"type":"sca", "arch":"x86_64"}'
------ response ------
"id":2377845297775805901,
```

After looking into the code, I think the cause is integration of jq (as a terminal highlighter?): https://github.com/openshift-online/ocm-cli/blob/90e6dd69c10143c0ac48b0ebad3e498a5a49951d/pkg/dump/dump.go#L31-L32

The loss with large numbers is a known issue to jq. Here's one of the most recent discussion: https://github.com/stedolan/jq/issues/1959
```
$ echo 4418489049307232906 | jq
4418489049307233000
```
This is really an edge case. I don't expect removing the jq integration because of this. However, will you consider adding an option for turning it off? so for rare cases like this, if a person does need the raw output, he can specify a flag to skip jq.

**Comment by cben** [comment]
Ooh, great find ❤️ ☹️ 

For an immediate workaround, pipe output to `| cat` this will supress colorization by `jq`.

We can look at other ways to colorize in-process.  But the benefit will go away if user pipes to jq, which realistically is a common wya to use ocm.
Javascript also has trouble with full int64 range.
Maybe we can re-think the API...

**Comment by cben** [comment]
Can you make that id a string?  That will help various clients...

IEEE doubles retain integer precision up to 2**53.  Is 2**64 really the range you need, or might they grow even bigger?
(BTW, is it opaque to the user - as most our fields named `id`?)

**Comment by petli-openshift** [comment]
unfortunately, it's not us that create these. We merely pass the data through from Red Hat Subscription Management to the client. This will unlikely cause any issue since the API will be mostly used by a specific client app, which does not use ocm-cli.
Just wanted to give a heads up here. There's really no need change anything because of it. It could be considered with other improvements related to the jq usage in ocm.

**Comment by jhernand** [comment]
This issue is addressed in #315.

**Comment by georgettica** [comment]
Added a note if a jq alternative in #315 , which is https://github.com/itchyny/gojq

I hope the extra info helps

**Comment by jhernand** [comment]
Thanks for the input @georgettica. This issue was solved using [nwidger/jsoncolor](https://github.com/nwidger/jsoncolor). The library that you mention is very interesting if we ever need additional `jq` functionallity. But for now we only need coloring, and for that `nwidger/jsoncolor` is enough.

**Comment by jhernand** [comment]
This has been fixed and included in [release 0.1.57](https://github.com/openshift-online/ocm-cli/releases/tag/v0.1.57).
